Add explanation for the error messages (#91)
OpenLiberty/guide-testcontainers: README.adoc

mvn liberty:devc -DcontainerRunOpts="-e DB_HOSTNAME=172.17.0.2" -DserverStartTimeout=240
```

> Wait a moment for dev mode to start. Some error messages are expected as a result from building the docker image. This is because, in order to print these messages, they are included on the standard error stream. However, in this case, they are not errors, just logs of the progress of this docker build. After you see the following message, your Liberty instance is ready in dev mode:

Reviewer: ```suggestion
Wait a moment for dev mode to start. Some error messages are expected as a result of building the docker image. Although these messages are included on the standard error stream, in this case they are not errors, just logs of the docker build progress. After you see the following message, your Liberty instance is ready in dev mode:
```
